docs/rules/attribute-names.md

```diff
@@ -33,6 +36,45 @@ The following patterns are not warnings:
 @property({attribute: 'camel-case-name'})
 camelCaseName: string;
 
+@property({attribute: 'camel-case-other-name'})
```

Comment:
this and the one above are warnings with the default configuration right?

since both should be `camelcasename` as an attribute

Reply:
No I don't think: https://github.com/43081j/eslint-plugin-lit/blob/ba7794c61c6198f38648b0baa7867407f277c8b1/src/test/rules/attribute-names_test.ts#L51

Current check is only to check that lower cased attribute is the same as attribute (so only check there are no upper cased char)
```ts
              #if (
                propConfig.attributeName.toLowerCase() !==
                propConfig.attributeName
              ) {
```
https://github.com/43081j/eslint-plugin-lit/blob/ba7794c61c6198f38648b0baa7867407f277c8b1/src/rules/attribute-names.ts#L53

Adding a contraint between attribute name and property name only appear in this PR

To enforce attribute value to be `camelcasename` in that case, you will need this PR with option `{"style": "none"}`